core/controllers/suggestion.py

```diff
@@ -148,6 +148,24 @@ def post(self):
 
         self.render_json(self.values)
 
+    @acl_decorators.can_update_suggestions
+    def put(self, suggestion_id):
+        """Handles POST requests.
```

Comment:
PUT* requests. In general, when you're copy-and-pasting similar code, please take a pass on the new code to make sure it's up-to-date, including any comments, pydocs, etc.

Reply:
Done